src/Raven.Client/ServerWide/Operations/Migration/OfflineMigrationConfiguration.cs

```diff
@@ -8,6 +8,10 @@ namespace Raven.Client.ServerWide.Operations.Migration
 {
     public class OfflineMigrationConfiguration
     {
+        private const string StorageExporterExecutable = "Raven.StorageExporter.exe";
+        private const string EsentDataFile = "Data.jfm";
```

Comment:
The Esent filename is supposed to be just `Data`, no?

Reply:
![image](https://user-images.githubusercontent.com/1250431/47136128-059db280-d2b3-11e8-9f5e-4b6a90fbeeaa.png)

I can see data and data.jfm. I used data.jfm as it is has better selectivity then simply *data*. Anyway I can change that if you wish.

Reply:
I just saw conversation on youtrack. I will change this to Data or Data.ravenfs

Reply:
Fixed